src/pages/InstallExtension.js

```diff
@@ -1,42 +1,35 @@
-import { LogoOnTop } from "../components/FormPage.sc";
-import strings from "../i18n/definitions";
-import { PageBackground, ExtensionContainer } from "./ExtensionInstalled.sc";
-import * as s from "./InstallExtension.sc";
+import InfoPage from "./info_page_shared/InfoPage";
+import Header from "./info_page_shared/Header";
+import Heading from "./info_page_shared/Heading";
+import Main from "./info_page_shared/Main";
+import ButtonContainer from "./info_page_shared/ButtonContainer";
+import Footer from "./info_page_shared/Footer";
+import Button from "./info_page_shared/Button";
+import { getExtensionInstallationLinks } from "../utils/misc/extensionCommunication";
+import MainImage from "./info_page_shared/MainImage";
+import FileDownloadOutlinedIcon from "@mui/icons-material/FileDownloadOutlined";
 
 export default function InstallExtension() {
   return (
-    <PageBackground>
-      <LogoOnTop />
-      <ExtensionContainer>
-        <s.InstallExtensionWrapper>
-          <h1>{strings.userCreated}</h1>
-          <h4>{strings.installExtension}</h4>
-          <p>{strings.extensionDescription}</p>
-          <p>{strings.extensionFunctionality}</p>
-          <ul>
-            <li>
-              <p>{strings.extensionAdvantage1}</p>
-            </li>
-            <li>
-              <p>{strings.extensionAdvantage2}</p>
-            </li>
-            <li>
-              <p>{strings.extensionAdvantage3}</p>
-            </li>
-          </ul>
-          <s.LinkContainer>
-            <s.OrangeButton>
-              <a href="https://chrome.google.com/webstore/detail/the-zeeguu-reader/ckncjmaednfephhbpeookmknhmjjodcd">
-                Install for Chrome or Edge
-              </a>
-            </s.OrangeButton>
-            <s.OrangeButton>
-              <a href="https://addons.mozilla.org/en-US/firefox/addon/the-zeeguu-reader/">Install for Firefox</a>
-            </s.OrangeButton>
-            <a href="/articles">Don't want the extension? Go to articles.</a>
-          </s.LinkContainer>
-        </s.InstallExtensionWrapper>
-      </ExtensionContainer>
-    </PageBackground>
+    <InfoPage>
+      <Header>
+        <Heading>You're almost there</Heading>
+      </Header>
+      <Main>
+        <p>
+          Time to install The Zeeguu Reader browser extension, which enables you
+          to&nbsp;read and translate articles and solve exercises
+        </p>
+        <MainImage src={"../static/images/find-extension.png"} />
```

Comment:
this page is not a modal, and it still uses a MainImage. What does this mean? Do we move some of the components out of the modal_shared into another folder, something like `user_dialog_shared`? or straight into the components folder?

Reply:
The `MainImage` is the **img** component that can be inserted inside any element (e.g., modal or page). It makes the image adapt its width (keeping its aspect ratio) to the width of its parent `<div>` (in our case, it's mostly used in the `<main>` sections of modals and new info pages). The Info Page had its own `MainImage`, separate from the one placed in the `modal_shared` folder, but I have noticed that both of them behaved identically, so I've followed the second solution, leaving only one `MainImage` and placed it in the `components` folder. <br><br>Perhaps we could also consider renaming it to `FullWidthImage`? What do you think?

Reply:
Good idea to rename to `FullWidthImage`. I think the name is very clear.